app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -33,8 +29,7 @@ def proof(
         should_proof_state_id:,
         threatmetrix_session_id:,
         timer:,
-        user_email:,
-        double_address_verification: false
+        user_email:
```

Comment:
do we not need `ipp_enrollment_in_progress` here?

Reply:
Soooo, my open question at the top is more along the lines of "do we need ipp_enrollment_in_progress at all, since I'm pretty sure it's always true? Except when an enrollment is failed or completed it's always in progress or establishing, right?"